src/compiler/diagnosticMessages.json

```diff
@@ -1035,6 +1035,10 @@
         "category": "Error",
         "code": 1356
     },
+    "An enum member name must be followed by a ',' or '='.": {

```

Comment:
Do you think this list should include a `}` because that should be allowed as a part of closing up the enum?

Reply:
Sure. I think the ideal would be something that implies it can be continued within the parent declaration (so some reader wouldn't think we're implying not to use comments)... but +1 until someone thinks of phrasing that works there.